index.js

```diff
@@ -1,7 +1,6 @@
 import delay from 'yoctodelay';
 
 export default async function pMinDelay(promise, minimumDelay, {delayRejection = true} = {}) {
-	const delayPromise = delay(minimumDelay);
-	await (delayRejection ? delayPromise : Promise.all([promise, delayPromise]));
+	await (delayRejection ? delay(minimumDelay) : Promise.resolve(promise));
```

Comment:
I don't think this is correct. When `delayRejection: false`, it will no longer delay the promise.

Reply:
For the fast fail scenario - i.e. `delayRejection: false` - it's already verified through this unit test https://github.com/sindresorhus/p-min-delay/blob/main/test.js#L35

Reply:
The tests may not cover every scenerio.